localisation/00_wc_job_titles.csv

```diff
@@ -63,3 +38,318 @@ job_spiritual_void_group;Shadow Priest;Pr
+job_marshal_tauren_group;Brave;;;;;;;;;;;;;x
+job_marshal_female_tauren_group;Brave;;;;;;;;;;;;;x
+job_treasurer_tauren_group;Master;;;;;;;;;;;;;x
+job_treasurer_female_tauren_group;Master;;;;;;;;;;;;;x
+job_spymaster_tauren_group;Chief Scout;;;;;;;;;;;;;x
+job_spymaster_female_tauren_group;Chief Scout;;;;;;;;;;;;;x
+job_chancellor_titanforged_group;Keeper;;;;;;;;;;;;;x
+job_chancellor_female_titanforged_group;Keeper;;;;;;;;;;;;;x
+job_marshal_titanforged_group;Protector;;;;;;;;;;;;;x
+job_marshal_female_titanforged_group;Protector;;;;;;;;;;;;;x
+job_treasurer_titanforged_group;Architect;;;;;;;;;;;;;x
+job_treasurer_female_titanforged_group;Architect;;;;;;;;;;;;;x
+job_spymaster_titanforged_group;Seeker;;;;;;;;;;;;;x
+job_spymaster_female_titanforged_group;Seeker;;;;;;;;;;;;;x
+job_chancellor_trogg_group;Boss;;;;;;;;;;;;;x
+job_chancellor_female_trogg_group;Boss;;;;;;;;;;;;;x
+job_marshal_trogg_group;Warrior;;;;;;;;;;;;;x
+job_marshal_female_trogg_group;Warrior;;;;;;;;;;;;;x
+job_treasurer_trogg_group;Drudge;;;;;;;;;;;;;x
+job_treasurer_female_trogg_group;Drudge;;;;;;;;;;;;;x
+job_spymaster_trogg_group;Scout;;;;;;;;;;;;;x
+job_spymaster_female_trogg_group;Scout;;;;;;;;;;;;;x
+job_chancellor_troll_group;Overseer;;;;;;;;;;;;;x
+job_chancellor_female_troll_group;Overseer;;;;;;;;;;;;;x
+job_marshal_troll_group;Warlord;;;;;;;;;;;;;x
+job_marshal_female_troll_group;Warlord;;;;;;;;;;;;;x
+job_treasurer_troll_group;Elder;;;;;;;;;;;;;x
+job_treasurer_female_troll_group;Elder;;;;;;;;;;;;;x
+job_spymaster_troll_group;Master;;;;;;;;;;;;;x
+job_spymaster_female_troll_group;Master;;;;;;;;;;;;;x
+job_chancellor_tuskarr_group;Boss;;;;;;;;;;;;;x
+job_chancellor_female_tuskarr_group;Boss;;;;;;;;;;;;;x
+job_marshal_tuskarr_group;Champion;;;;;;;;;;;;;x
+job_marshal_female_tuskarr_group;Champion;;;;;;;;;;;;;x
+job_treasurer_tuskarr_group;Elder;;;;;;;;;;;;;x
+job_treasurer_female_tuskarr_group;Elder;;;;;;;;;;;;;x
+job_spymaster_tuskarr_group;Trapper;;;;;;;;;;;;;x
+job_spymaster_female_tuskarr_group;Trapper;;;;;;;;;;;;;x
+job_chancellor_undead_group;Ascendant;;;;;;;;;;;;;x
+job_chancellor_female_undead_group;Ascendant;;;;;;;;;;;;;x
+job_marshal_undead_group;Blade of the Damned;;;;;;;;;;;;;x
+job_marshal_female_undead_group;Blade of the Damned;;;;;;;;;;;;;x
+job_treasurer_undead_group;Soul Harvester;;;;;;;;;;;;;x
+job_treasurer_female_undead_group;Soul Harvester;;;;;;;;;;;;;x
+job_spymaster_undead_group;Chief Apothecary;;;;;;;;;;;;;x
+job_spymaster_female_undead_group;Chief Apothecary;;;;;;;;;;;;;x
+job_chancellor_vrykul_group;Overthane;;;;;;;;;;;;;x
+job_chancellor_female_vrykul_group;Overthane;;;;;;;;;;;;;x
+job_marshal_vrykul_group;Warlord;;;;;;;;;;;;;x
+job_marshal_female_vrykul_group;Warlord;;;;;;;;;;;;;x
+job_treasurer_vrykul_group;Constructor;;;;;;;;;;;;;x
+job_treasurer_female_vrykul_group;Constructor;;;;;;;;;;;;;x
+job_spymaster_vrykul_group;Great Hunter;;;;;;;;;;;;;x
+job_spymaster_female_vrykul_group;Great Huntress;;;;;;;;;;;;;x
```

Comment:
Up to you, but I would keep **_Spymaster_**. Hunter, making and revealing plots, sounds weird.